src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthNukleus.java

```diff
@@ -101,10 +103,20 @@ private void onResolve(
         final long correlationId = resolve.correlationId();
         final String realm = resolve.realm().asString();
 
+        String issuer = null;
+        String audience = null;
+        if(resolve.extension().sizeof() > 0)
+        {
+            final OAuthResolveExFW resolveExtension =
+                    resolveROEx.wrap(buffer, resolve.extension().offset(), resolve.extension().limit());
```

Comment:
Consider using `tryWrap` instead of `wrap` to avoid exception when input is malformed, rename `resolveExtension` to `resolveEx` and handle case where `resolveEx` is `null`.